### PR TITLE
Remove closing php tags

### PR DESCRIPTION
**Pull request by vishalkakadiya**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -333,9 +333,9 @@ function twentysixteen_hex2rgb( $color ) {
 	$color = trim( $color, '#' );
 
 	if ( strlen( $color ) === 3 ) {
-		$r = hexdec( substr( $color, 0, 1 ).substr( $color, 0, 1 ) );
-		$g = hexdec( substr( $color, 1, 1 ).substr( $color, 1, 1 ) );
-		$b = hexdec( substr( $color, 2, 1 ).substr( $color, 2, 1 ) );
+		$r = hexdec( substr( $color, 0, 1 ) . substr( $color, 0, 1 ) );
+		$g = hexdec( substr( $color, 1, 1 ) . substr( $color, 1, 1 ) );
+		$b = hexdec( substr( $color, 2, 1 ) . substr( $color, 2, 1 ) );
```
Comment (ntwb) [inline functions.php:338]: The above 3 changes should not be in this pull request, can you remove these please @vishalkakadiya

File: template-parts/content-search.php [hunk 2]
```diff
@@ -50,4 +50,3 @@
 
 	<?php endif; ?>
 </article><!-- #post-## -->
-
```
Comment (ntwb) [inline template-parts/content-search.php:53]: Why is this blank line removed?

**Review by ntwb — CHANGES_REQUESTED** [review]
I've added a couple of requests/questions.

**Comment by ntwb** [comment]
Note: Please do not merge this PR until a decision/consensus has been agreed upon for what themes to make these changes to as part of https://core.trac.wordpress.org/ticket/40039

File: functions.php (hunk 1 above)
Comment (vishalkakadiya) [reply functions.php:338]: @ntwb  I am removing this but, without fixing this **travis** build was **failed**, so, that's why I fixed it and travis pass the built. But don't worry removing it.

Comment (ntwb) [reply functions.php:338]: Thanks, can you create a separate pull request with this change then please, then add a link to it on the #498 PR, please, and thank you

**Comment by ntwb** [comment]
This pull request is now superseded by #498, @vishalkakadiya please close this pull request

**Pull request closed without merging**